bundles/org.eclipse.passage.loc.users.ui/src/org/eclipse/passage/loc/internal/users/ui/SelectUser.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.loc.internal.users.ui;
+
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.eclipse.e4.core.contexts.IEclipseContext;
+import org.eclipse.passage.lic.jface.resource.LicensingImages;
+import org.eclipse.passage.lic.users.UserDescriptor;
+import org.eclipse.passage.lic.users.UserOriginDescriptor;
+import org.eclipse.passage.lic.users.model.meta.UsersPackage;
+import org.eclipse.passage.lic.users.registry.UserRegistry;
+import org.eclipse.passage.loc.internal.users.ui.i18n.UsersUiMessages;
+import org.eclipse.passage.loc.internal.workbench.SelectRequest;
+import org.eclipse.passage.loc.internal.workbench.SupplySelectRequest;
+import org.eclipse.passage.loc.jface.dialogs.Appearance;
+import org.eclipse.passage.loc.users.core.Users;
+
+/**
+ * Selects or creates {@link UserOriginDescriptor}. Will return either
+ * {@link Optional} with selected/created license plan or
+ * {@link Optional#empty()}
+ * 
+ * @since 0.6
+ *
+ */
+public final class SelectUser extends SupplySelectRequest<UserDescriptor> {
+
+	public SelectUser(IEclipseContext context) {
+		super(context);
+	}
+
+	@Override
+	public SelectRequest<UserDescriptor> get() {
+		return new SelectRequest<>(UserDescriptor.class, domain(), input(), appearance());
+	}
+
+	private Supplier<Iterable<UserDescriptor>> input() {
+		return () -> StreamSupport.stream(context.get(UserRegistry.class).getUsers().spliterator(), false)//
```

Comment:
Can we please format `stream` sequences like 
 ```java
source.stream() //
   .oprtation1() //
   .operation2() //
   .terminalOperation();
```

Reply:
It is there